core/start-blueos-core

```diff
@@ -148,9 +149,56 @@ function create_service {
     done
 
     # Use run_service to start the service with the memory limit
-    tmux send-keys -t $SESSION_NAME "run-service '$command' $memory_limit_mb" C-m
+    tmux send-keys -t $SESSION_NAME "run-service '$SERVICE_NAME' '$command' $memory_limit_mb " C-m
 }
 
+ssh_command() {
+  ssh -i /root/.config/.ssh/id_rsa -o StrictHostKeyChecking=no pi@localhost "$1"
+}
+
+prepare_cgroups() {
```

Comment:
We should test the ssh_command is working, if not, we should abort this process and continue the start script.
The reason behind this is that the start script is the root of the entire BlueOS, it can't fail.

Reply:
done